community/server/src/main/java/org/neo4j/server/ServerBootstrapper.java

```diff
@@ -172,6 +172,7 @@ protected abstract NeoServer createNeoServer( Config config, GraphDatabaseDepend
     private static LogProvider setupLogging( Config config )
     {
         LogProvider userLogProvider = FormattedLogProvider.withoutRenderingContext()
+                            .withZoneId( config.get( GraphDatabaseSettings.log_timezone ).getZoneId() )
```

Comment:
The field `log` is assigned a non-timezone logger first, not sure if this will pose a problem or not.

Reply:
That is for cases where we fail to parse provided configuration while we still need to provide log to the user, for those cases we can't provide valid configuration of logger and just use defaults